docker/dev/docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+version: '3.0'
+
+services:
+  backend:
+    build:
+      context: ../../
+      dockerfile: docker/dev/Dockerfile
+    image: soc-backend:sandbox
+    volumes:
```

Comment:
I think the devs should be informed that they need an existing logs file in the parent-parent directory. Or is one created for them?

Reply:
After testing it, I have found out that the logs folder is made automatically by docker itself.

Reply:
Cool